styles.css

```diff
@@ -1781,7 +1781,7 @@ a {
   }
 
   .popup_description .popup_text {
-    padding-bottom: 3rem;
+    padding-bottom: rem;
```

Comment:
- [ ] Although your pop up can't be seen now due to the bug we spotted earlier, you seem to have done this in a hurry and it appears you only have an empty value for `padding-bottom` here. Kindly remove it or include a value if you intend to retain it in your styling :pleading_face: